[Term Entry] Kotlin Math .sqrt() (#3676)
Codecademy/docs: content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md

---

> Title: 'Math.sqrt()'

Reviewer: ```suggestion
Title: 'sqrt()'
```

Description: 'Calculates the square root of a given number.'
Subjects:

>   - 'Mathematics'
>   - 'Programming'

Reviewer: ```suggestion
  - 'Code Foundations'
```

  - 'Computer Science'
Tags:
  - 'Math'
  - 'Functions'
CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---

> The **`Math.sqrt()`** method in Kotlin's `Math` class is used to calculate the square root of a given number.

Reviewer: ```suggestion
The **`math.sqrt()`** method in Kotlin's `math` class is used to calculate the square root of a given number.
```

## Syntax

```pseudo

> val squareRoot = Math.sqrt(number)

Reviewer: ```suggestion
import kotlin.math.sqrt
val squareRoot = sqrt(number)
```

```

> - `number`: The number for which you want to calculate the square root. It should be a non-negative value (zero or a positive number).

Reviewer: ```suggestion
- `number`: The number for which you want to calculate the square root. It should be a non-negative value (zero or a positive number) of type `Double` or `Float`.
```

> The `Math.sqrt()` method returns a value of type `Double` representing the square root of the input number. The result is a non-negative value or NaN if the input is negative.

Reviewer: ```suggestion
The method returns a number, of the same type given, representing the square root. The result is a non-negative value or `NaN` if the input is negative.
```

## Example

> This example shows how to use the `Math.sqrt()` method to calculate the square root of a number:

Reviewer: ```suggestion
This example shows how to use the `sqrt()` method to calculate the square root of a number:
```

> ```kotlin

Reviewer: ```suggestion
```kotlin
import kotlin.math.sqrt
```

fun main() {
    val number = 25.0

>     val squareRoot = Math.sqrt(number)

Reviewer: ```suggestion
    val squareRoot = sqrt(number)
```
